tendrl/node_agent/manager/manager.py

```diff
@@ -105,6 +105,7 @@ def __init__(self, machine_id):
         ).__init__(
             "node",
             utils.get_local_node_context(),
+            utils.get_local_node_context(),
```

Comment:
One is used as integration_context and other as node_id. They are used for two different purposes and as Manager constructor needs both, using it twice to pass same value for two different purposes.